pyrevitlib/pyrevit/compat.py

```diff
@@ -8,11 +8,15 @@
 """
 
 import sys
+import System
 
 PY2 = sys.version_info[0] == 2
 PY3 = sys.version_info[0] == 3
-IRONPY2712 = sys.version_info[:3] == (2, 7, 12)
-IRONPY340 = sys.version_info[:3] == (3, 4, 0)
+IRONPY = '.net' in sys.version.lower()
+IRONPY2 = PY2 and '.net' in sys.version.lower()
```

Comment:
to avoid repetition and improve clarity:
IRONPY2 = IRONPY and PY2
IRONPY3 = IRONPY and PY3

Reply:
@sanzoghenzo I've revised this